sql-cli/sql_cli/settings.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+
+from airflow.configuration import conf
+
+# This needs to be updated with the `astro dev init` command to set the appropriate project directory.
+SQL_CLI_PROJECT_DIRECTORY = Path(
+    conf.get("astro_sdk", "sql_cli_project_directory", fallback="../tests/")
```

Comment:
Since we are trying to abstract Airflow, other options to achieve the same could be:
-  pass a parameter on the command line
- default to the "current directory" from where the user is running the command
Without the need to set an Airflow variable.

Reply:
This might need some more thought. Can I work on this later in another PR once the scaffolding/init command and astro-cli integration progresses a bit more how to mount/use the project directory in the docker container? I am a bit unclear at the moment.